src/constants/urls.ts

```diff
@@ -0,0 +1,2 @@
+export const DISCORD_INVITE_LINK = 'https://discord.com/invite/QFkNsjTkzD'
+export const LIQUIDITY_V3_LINK = 'https://swapr.liquidity.eth.limo/'
```

Comment:
This link goes to swapbox and we want it to go to the pools and TVL metrics I would say ( `/info/pools` )

```suggestion
export const LIQUIDITY_V3_LINK = 'https://swapr.liquidity.eth.limo/#/info/pools'
```